nats/aio/client.py

```diff
@@ -219,7 +219,7 @@ def __init__(self) -> None:
 
     async def connect(
         self,
-        servers: List[str] = ["nats://127.0.0.1:4222"],
+        servers: Union[str, List[str]] = ["nats://127.0.0.1:4222"],
```

Comment:
btw, what do you think about adding another commit so that both the `connect` methods use the same default, either `localhost` or `127.0.0.1`?

Reply:
Thanks, maybe let's use `localhost` for both? wdyt?

Reply:
Agreed, this way we are not defaulting to IPv4. I'll add another commit to address that.

Reply:
ok ready :+1: